lib/bundler/bundler.gemspec

```diff
@@ -34,14 +34,11 @@ Gem::Specification.new do |s|
   s.required_ruby_version     = ">= 2.3.0"
   s.required_rubygems_version = ">= 2.5.2"
 
-  # s.files = Dir.glob("{lib,man,exe}/**/*", File::FNM_DOTMATCH).reject {|f| File.directory?(f) }
+  s.files = (Dir.glob("lib/bundler/**/*", File::FNM_DOTMATCH) + Dir.glob("man/bundler*") + Dir.glob("libexec/bundle*")).reject {|f| File.directory?(f) }
 
-  # Include the CHANGELOG.md, LICENSE.md, README.md manually
-  # s.files += %w[CHANGELOG.md LICENSE.md README.md]
-  # include the gemspec itself because warbler breaks w/o it
-  s.files += %w[bundler.gemspec]
+  s.files += ["lib/bundler.rb"]
 
-  s.bindir        = "exe"
+  s.bindir        = "libexec"
```

Comment:
Do you have an idea that `files` and `bindir` variables integrate with the upstream gemspec?

Reply:
I thought a bit about it, but I couldn't come up with a good solution. Let's see how it goes, but I don't expect the gemspec to change much, so there's shouldn't be many conflicts. If this doesn't work for you, we can come up with another solution.